### PR TITLE
Youbora: add youbora support to ios

### PR DESCRIPTION
**Pull request by nirkatz2020**
1. initiate the plugins in the `playerOptions`.
2. Update the player plugins (currently only youbora) in the load method.

### CODE REVIEW

File: ios/KalturaPlayerViewManager.swift [hunk 1]
```diff
@@ -400,4 +410,27 @@ class KalturaPlayerRNView : UIView {
...
+            let youboraConfig = AnalyticsConfig(params:youboraParams)
+            pluginConfigs[YouboraPlugin.pluginName] = youboraConfig
+        }
+        if (!pluginConfigs.isEmpty){
+            return PluginConfig(config: pluginConfigs)
+        }
+        return nil
+    }
+    
+    func updatePluginsConfig(plugins: Dictionary<String, Any>) {
```
Comment (djinely) [inline ios/KalturaPlayerViewManager.swift:426]: is this function necessary? can't we use updateYouboraConfig directly?

File: ios/KalturaPlayerViewManager.swift [hunk 2]
```diff
@@ -313,6 +313,12 @@ class KalturaPlayerRNView : UIView {
         playerOptions.preload = options["preload"] as! Bool
         playerOptions.autoPlay = options["autoplay"] as! Bool
         playerOptions.ks = options["ks"] as? String
+
+        if let plugins = options["plugins"] as? Dictionary<String, Any> {
+            if let pluginsConfig =  getPluginsConfigs(plugins:plugins ) {
```
Comment (djinely) [inline ios/KalturaPlayerViewManager.swift]: please remove unnecessary spaces

**Review by djinely — APPROVED** [review]

File: ios/KalturaPlayerViewManager.swift (hunk 1 above)
Comment (nirkatz2020) [reply ios/KalturaPlayerViewManager.swift:426]: Yes, there should be more plugin configurations that will be added later on.

File: ios/KalturaPlayerViewManager.swift (hunk 2 above)
Comment (nirkatz2020) [reply ios/KalturaPlayerViewManager.swift]: done

**Pull request merged**